apps/dot_ext/forms.py

```diff
@@ -127,7 +127,11 @@ def clean_redirect_uris(self):
 
 
 class SimpleAllowForm(DotAllowForm):
-    pass
+    code_challange = forms.CharField(required=False, widget=forms.HiddenInput())
```

Comment:
Needing to do a string replace across the changed files for "challange" -> "challenge" to correct spelling.

Reply:
@dtisza1 done!